src/IssueViz/Resources.resx

```diff
@@ -138,4 +138,7 @@
   <data name="IssueVisualizationToolWindowCaption" xml:space="preserve">
     <value>SonarLint Issue Visualization</value>
   </data>
+  <data name="ERR_QuickFixes_Exception" xml:space="preserve">
+    <value>[Quick fixes] Error processing quick fixes: {0}</value>
```

Comment:
I used the same message for both `catch` blocks. The call stack will tell us where it occured.